textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -42,21 +44,57 @@
     private List<Attribute> searchInAttributes;
     private ITuple sourceTuple;
     private Schema returnSchema;
+    private NlpTokenType inputNlpTokenType = null;
+    private String flag = null;
```

Comment:
Maybe you missed or disagreed with my earlier comment.  Do you think it's better to rename "flag" to something more meaningful?